requirements.txt

```diff
@@ -1,14 +1,13 @@
 datetime
 pyfiglet
 PyYAML>=5.1
-git+https://github.com/powerfulseal/powerfulseal.git@3.3.0
```

Comment:
Wonder if there's anything in the requirements that we were having powerfulseal install and we didn't out right define in this file. Will have to keep an eye out on any missing requirements for all the different scenarios as we go forward with adding the plugins

Reply:
Yes, just wanted to note it in case something starts breaking